omnibus-sepolia-carina.toml

```diff
@@ -1,510 +0,0 @@
-name = "synthetix-omnibus"
```

Comment:
please delete all files related to carina. In e2e/tests/, and tomls/, and search places like package.json for any mention of carina.